package.json

```diff
@@ -44,13 +51,17 @@
     "@types/node": "^18",
     "@types/react": "^18.0.37",
     "@types/react-dom": "^18.0.11",
-    "@typescript-eslint/eslint-plugin": "^5.59.0",
-    "@typescript-eslint/parser": "^5.59.0",
+    "@typescript-eslint/eslint-plugin": "latest",
+    "@typescript-eslint/parser": "latest",
```

Comment:
semantic versioning is preferred over the `latest`

Reply:
done